VERSION

```diff
@@ -1 +1 @@
-0.61.0
+0.61.1 
```

Comment:
Newline at the end of the file, the tests are complaining

Reply:
added 👌